packages/eslint-plugin-nx/src/configs/react-jsx.ts

```diff
@@ -63,6 +63,7 @@ export default {
      * React Hooks rule configurations
      * https://github.com/facebook/react/tree/master/packages/eslint-plugin-react-hooks
      */
+    'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Hm this is already set to `warn` in the `react-base` config. However, it's overwritten in `react-typescript` :thinking:

That doesn't seem right to me... In fact, I think we should extend the facebook config.

Can you remove hooks rules from here as well as `react-typescript` and `react-base` then add `extends: ["plugin:react-hooks/recommended"]` to `react-base`?